### PR TITLE
Fix error when task BuildTools is execute

### PR DESCRIPTION
**Pull request by Romaindu35**
image

### CODE REVIEW

File: mainPlugin/src/main/java/fr/il_totore/manadrop/spigot/task/BuildTools.java [hunk 1]
```diff
@@ -22,10 +22,13 @@
     public boolean showMavenInstallCheckLogs = false;
     public boolean showBuildToolsLogs = true;
     public int maxRamAllowed = 1024;
-    public String mavenPath = System.getenv("MAVEN_HOME") + "/bin/mvn" + (System.getenv("OS") != null && System.getenv("OS").toLowerCase().contains("win") ? ".cmd" : "");
+    public String mavenPath = System.getProperty("user.home") + ".m2/repository/";
```
Comment (Iltotore) [inline mainPlugin/src/main/java/fr/il_totore/manadrop/spigot/task/BuildTools.java]: This doesn't solve anything. What if you're maven installation is not in `~/.m2/repository` ? Also this is the **repository** dir. Not the executable's one.

**Review by Iltotore — CHANGES_REQUESTED** [review]

File: mainPlugin/src/main/java/fr/il_totore/manadrop/spigot/task/BuildTools.java (hunk 1 above)
Comment (Romaindu35) [reply mainPlugin/src/main/java/fr/il_totore/manadrop/spigot/task/BuildTools.java]: Fixed in new commit

**Comment by Iltotore** [comment]
This doesn't fix the issue quoted in the review. What if we install maven in another location ? What if we set the user's repository elsewhere ?

All you have to do is to setup correctly your MAVEN_PATH

**Comment by Iltotore** [comment]
Due to the lack of activity, I will close this PR. Feel free to create a new one if needed.

**Pull request closed without merging**